cove_bods/templates/cove_bods/additional_checks_table.html

```diff
@@ -25,10 +25,10 @@
         {% elif additional_check.type == 'entity_statement_out_of_order' %}
             <tr>
                 <td>
-                    {% trans 'This statement references an entity but that entity is defined after this statement.' %}
+                    {% blocktrans %}<code>Entity statement</code> not in correct order. Check that the Entity statement is placed in the array before any statement referencing it.{% endblocktrans %}
```

Comment:
I'm not sure how to trigger this error - the test data in https://github.com/openownership/lib-cove-bods/issues/131 triggers entity_statement_missing in the case where an entity statement is after the relationship statement. Is that the expected behaviour?

Reply:
'entity_statement_out_of_order' comes from the previous version of the code, and while it isn't going to be triggered for 0.4, I assume we don't want to be changing the behaviour for BODS version < 0.4 data (at least without thinking about it a bit).

Reply:
This error is generated for pre BODS 0.4 data. I've just checked. Like:

```
[

  {
    "statementID": "fbfd0547-d0c6-4a00-b559-5c5e91c34f5c",
    "statementType": "ownershipOrControlStatement",
    "isComponent": false,
    "statementDate": "2017-11-18",
    "subject": {
      "describedByEntityStatement": "1dc0e987-5c57-4a1c-b3ad-61353b66a9b7"
    },
    "interestedParty": {
      "describedByPersonStatement": "019a93f1-e470-42e9-957b-03559861b2e2"
    },
    "interests": [
      {
        "type": "shareholding",
        "interestLevel": "direct",
        "beneficialOwnershipOrControl":true,
        "startDate": "2016-04-06",
        "share": {
          "exact": 100,
          "minimum": 100,
          "maximum": 100
        }
      }
    ],
    "publicationDetails": {
        "publicationDate": "2018-02-13",
        "bodsVersion": "0.2",
        "publisher": {
            "name": "CHRINON LTD"
        }
    }
  },
  {
    "statementID": "1dc0e987-5c57-4a1c-b3ad-61353b66a9b7",
    "statementType": "entityStatement",
    "isComponent": false,
    "statementDate": "2017-11-18",
    "entityType": "registeredEntity",
    "name": "CHRINON LTD",
    "foundingDate": "2010-11-18",
    "identifiers": [
      {
        "scheme": "GB-COH",
        "id": "07444723"
      }
    ],
    "publicationDetails": {
        "publicationDate": "2018-02-13",
        "bodsVersion": "0.2",
        "publisher": {
            "name": "CHRINON LTD"
        }
    }
  },
  {
    "statementID": "019a93f1-e470-42e9-957b-03559861b2e2",
    "statementType": "personStatement",
    "isComponent": false,
    "statementDate": "2017-11-18",
    "personType": "knownPerson",
    "nationalities": [
      {
        "code": "GB"
      }
    ],
    "names": [
      {
        "type": "individual",
        "fullName": "Christopher Taggart",
        "givenName": "Christopher",
        "familyName": "Taggart"
      },
      {
        "type": "alternative",
        "fullName": "Chris Taggart"
      }
    ],
    "birthDate": "1964-04",
    "addresses": [
      {
        "type": "service",
        "address": "Aston House, Cornwall Avenue, London",
        "country": "GB",
        "postCode": "N3 1LF"
      }
    ],
    "publicationDetails": {
        "publicationDate": "2018-02-13",
        "bodsVersion": "0.2",
        "publisher": {
            "name": "CHRINON LTD"
        }
    }
  }
]
```